### PR TITLE
i18n(ja): Add new `aws.mdx` translation

### PR DESCRIPTION
**Pull request by piro-hiroki**
#### Description (required)

Translate `aws.mdx` into Japanese！

file: [guides/deploy/aws.mdx](https://github.com/withastro/docs/blob/main/src/content/docs/en/guides/deploy/aws.mdx)

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

**Comment by sarah11918** [comment]
Hi @piro-hiroki ! Just checking if this PR should still be marked as a draft?

If it's ready for editing, then please change the status so that it is ready for review! We are hoping to merge as many PRs by next week as we can before the docs site code migrates to Starlight. So if this is ready, we would like to be able to review it.  Thanks!

**Comment by sarah11918** [comment]
Hi @piro-hiroki ! Is this one ready for a review now?

File: src/content/docs/ja/guides/deploy/aws.mdx
Comment (morinokami) [inline src/content/docs/ja/guides/deploy/aws.mdx:188]: https://github.com/withastro/docs/pull/5511 was merged after this PR was created.

```suggestion

### CloudFrontのエラーページのセットアップ

S3はデフォルトで、ファイルが見つからなかった場合には404エラーを、ファイルがプライベートであった場合には403エラーを返します。いずれの場合でも、ユーザーは汚いXMLエラーページを見ることとなります。

これを変更するには、CloudFrontディストリビューションの**設定 > エラーページ**に**カスタムエラーレスポンス**を追加します。

1. 404エラーのためのカスタムエラーレスポンスを以下の値で設定します:
    * **HTTPエラーコード:** 404: Not Found
    * **エラーレスポンスをカスタマイズ:** Yes
    * **レスポンスページのパス:** `/index.html`
    * **HTTPレスポンスコード:** 200: OK

2. 403エラーのためのカスタムエラーレスポンスを以下の値で設定します:
    * **HTTPエラーコード:** 403: Forbidden
    * **エラーレスポンスをカスタマイズ:** Yes
    * **レスポンスページのパス:** `/index.html`
    * **HTTPレスポンスコード:** 200: OK

  :::tip
  ベストなユーザー体験のために、プロジェクトに[カスタム404エラーページを作成](/ja/core-concepts/astro-pages/#カスタム404エラーページ)し、**レスポンスページのパス**を`/404.html`にセットすることもできます。
  :::

```

File: src/content/docs/ja/guides/deploy/aws.mdx
Comment (morinokami) [inline src/content/docs/ja/guides/deploy/aws.mdx:191]:
```suggestion
AWSで継続的デプロイを設定する方法はたくさんあります。GitHub 上でホストされているコードの場合、[GitHub Actions](https://github.com/features/actions) を使ってコミットをプッシュするたびにウェブサイトをデプロイする方法があります。
```

File: src/content/docs/ja/guides/deploy/aws.mdx
Comment (morinokami) [inline src/content/docs/ja/guides/deploy/aws.mdx:219]:
```suggestion
    `<DISTRIBUTION_ARN>`と`<BUCKET_NAME>`を忘れずに置き換えてください。ARN は**CloudFront > ディストリビューション > 詳細**で確認できます。
```

File: src/content/docs/ja/guides/deploy/aws.mdx
Comment (morinokami) [inline src/content/docs/ja/guides/deploy/aws.mdx:222]:
```suggestion
2. 新しいIAMユーザーを作成し、そのユーザーにポリシーをアタッチします。これにより、`AWS_SECRET_ACCESS_KEY`と`AWS_ACCESS_KEY_ID`が提供されます。
```

File: src/content/docs/ja/guides/deploy/aws.mdx
Comment (morinokami) [inline src/content/docs/ja/guides/deploy/aws.mdx:224]:
```suggestion
3. 以下のサンプルワークフローをあなたのリポジトリの`.github/workflows/deploy.yml`に追加し、GitHubにプッシュしてください。`AWS_ACCESS_KEY_ID`、`AWS_SECRET_ACCESS_KEY`、`BUCKET_ID`、および `DISTRIBUTION_ID`を、GitHubリポジトリの**Settings** > **Secrets** > **Actions**から"secrets"として追加する必要があります。<kbd>New repository secret</kbd>をクリックして、各値を追加します。
```

File: src/content/docs/ja/guides/deploy/aws.mdx
Comment (morinokami) [inline src/content/docs/ja/guides/deploy/aws.mdx:257]:
```suggestion
    `BUCKET_ID`はS3バケットの名前です。`DISTRIBUTION_ID`はCloudFrontのディストリビューションIDです。CloudFrontのディストリビューションIDは**CloudFront > ディストリビューション > ID**で確認できます。
```

**Review by morinokami — CHANGES_REQUESTED** [review]
@sarah11918 Added 23 suggestions, phew😅 I think we can merge this PR once those suggestions are merged.

@piro-hiroki Please let me know if you have anything to add!

**Review by sarah11918 — APPROVED** [review]
Thank you both @piro-hiroki and @morinokami ! I'm so happy to get all these new pages translated in docs! :tada:

**Pull request merged**
